packages/generator-common/src/sdk-metadata/pregenerated-lib.spec.ts

```diff
@@ -39,6 +40,13 @@ describe('pregenerated-lib', () => {
     );
   }, 30000);
 
+  it('gives version for exisitng client', async () => {
+    nock('http://registry.npmjs.org/')
+      .get(/@sap-cloud-sdk\/core\/latest/)
+      .reply(200, { version: '1.2.3' });
+    await getLatestVersionOfNpmPackage('@sap-cloud-sdk/core');
```

Comment:
I know it does not fail, but having `expect` would be better.

Reply:
no this is a mistake. I forgot the assertion. I will change that.